samples/bluetooth/mesh/light_ctrl/sample.yaml

```diff
@@ -17,3 +17,19 @@ tests:
       nrf5340dk_nrf5340_cpuapp_ns thingy53_nrf5340_cpuapp thingy53_nrf5340_cpuapp_ns
       nrf21540dk_nrf52840 nrf52833dk_nrf52833
     tags: bluetooth ci_build
+  sample.bluetooth.mesh.light_ctrl.emds:
```

Comment:
emds should be tested on CI on regular base considering the last quality problems that customers claimed to us.

Reply:
EMDS has been tested, but the test has to be run manually. We should create the test to run in regression, but that is not a part of adding emds to rpl. EMDS by it self is a seperate part. We could create system test that test storage of rpl and all models by using emds, but that is not a test of emds. But a test of the models and rpl. EMDS can indirectly be tested by that.